low-level/sensors/impl/generic/node_hw.py

```diff
@@ -448,6 +463,62 @@ def run(self):
             logger.warning(f"{self.SENSOR_NAME} Node hw monitoring disabled")
             self.shutdown()
 
+    def _check_faulty_resource_status(self):
+        # Currently, the SSPL sensor relies on IPMI SEL events to track server
+        # resources fault & recovery. However, its been observed multiple times
+        # with Supermicro servers, that SEL not getting updated in case recoveries
+        # of some resources.
+        # So sensor additionally polling faulty or missing resources, in addition
+        # to current polling for SEL.
```

Comment:
Is there a chance to generate two alerts for same fault, one from this sensor and other one from IPMI SEL? In that case, I think we should not do it.

Reply:
No. we are processing this function only when the SEL list execution is complete.
We are checking those faulty resources that are now ok state but somehow the SEL entry is missed to update in ipmitool sel list.
For more detail can you please check https://jts.seagate.com/browse/EOS-19807 description?